### PR TITLE
feat: all current updates except plugins

### PR DESCRIPTION
**Pull request by philhallbio**
## Description

All current updates expect plugins

### CODE REVIEW

**Comment by joshuajames-smith** [comment]
@ChasNelson1990 am I right in thinking you set **DOCUMENT** repo to produce staging links?

**Comment by ChasNelson1990** [comment]
@joshuajames-smith @philhallbio the deploy action ran, but for some reason isn't showing here :confused:. However it failed:

```
Error:  Unable to build website for locale en.
Error:  Error: Docusaurus found broken links!
```

Did you build this locally first @philhallbio - did it build and server fine?

**Comment by philhallbio** [comment]
> @joshuajames-smith @philhallbio the deploy action ran, but for some reason isn't showing here 😕. However it failed:
> 
> ```
> Error:  Unable to build website for locale en.
> Error:  Error: Docusaurus found broken links!
> ```
>
> Did you build this locally first @philhallbio - did it build and server fine?

I didn't build locally first (not sure how) wrote locally and corrected anything that came up from PR checks.

**Comment by ChasNelson1990** [comment]
@joshuajames-smith - @philhallbio and I have given you a magic staging link :-)

File: docs/basics/firststeps.md [hunk 1]
```diff
@@ -20,11 +20,15 @@ To delete the current annotation in progress, click the **Clear Annotation** but
 To select a different annotation click the **Select** button, click an area of the desired annotation and select the desired functionality to alter.
 To undo the last action, click the **Undo last action** and to redo the action, click the **Redo last action**.
 
-Once annotations have been completed, click the **Save Annotations** button to confirm your changes.
-Please note: if you leave annotate without saving then your annotations will not be added to the image metadata.
+At any point, click the **Save Annotations** button to save your progress.
```
Comment (ChasNelson1990) [inline docs/basics/firststeps.md:31]: Maybe in the image above we could add some tooltips/text with arrows saying which button is which?

File: docs/basics/firststeps.md [hunk 2]
```diff
@@ -20,11 +20,15 @@ To delete the current annotation in progress, click the **Clear Annotation** but
...
+_Tip: Save regularly as, if you leave annotate or switch images without saving, you will lose any unsaved annotations._
+
+Once annotations have been completed, click the **Mark Annotation As Complete** button.
```
Comment (ChasNelson1990) [inline docs/basics/firststeps.md:36]: We need an image showing this button

File: docs/basics/paintbrush.md [hunk 3]
```diff
@@ -37,3 +37,8 @@ Clicking the **Use 3D brushstrokes** button will activate the 3D brush, 3D annot
 The transparency of active and non-active annotations can be altered by clicking the **Annotation Transparency** button.
 Using the two sliders allows you to modify the transparency of both the current (brush) and all other (brush) annotations in the range of 0-100% (opacity).
 This can be a very useful tool to determine the accuracy of drawn annotations by comparing to the background image.
+
+## Strokes vs. Pixels
+
+Brush strokes can be converted from the standard vector-graphic representation into pixel level annotations, based on the size of the pixels of the image being annotated.
+To convert, click the **Show strokes as pixels** button.
```
Comment (ChasNelson1990) [inline docs/basics/paintbrush.md:47]: Either we need a specific image of this button, or some text labels on the summary image at the top of the page

File: docs/getstarted/createproject.md [hunk 4]
```diff
@@ -19,20 +19,20 @@ A project can be opened either to the Curate interface, by clicking the **Open X
...
 Now that a project has been created, you can begin to upload data.
 
-![Picture Missing]
+![Project Management Page](/img/manage/manage_projects.png)
 
 Open the project by clicking the specific **Open X in Curate** button for the desired project.
 This will load the Curate interface where you can begin uploading images.
 
-![Picture missing]
+![Curate Upload Box]
```
Comment (ChasNelson1990) [inline docs/getstarted/createproject.md]: Does not work

File: docs/getstarted/createproject.md [hunk 5]
```diff
@@ -19,20 +19,20 @@ A project can be opened either to the Curate interface, by clicking the **Open X
...
 
 In Curate, click the **Upload Image** button.
 This will open the standard file navigation pop-up for your system.
 Navigate to the desired image(s) and select all those for upload.
 
 Click the **Open** (or equivalent) button to begin the upload.
 
-![Picture missing]
+![Progress Bar]
```
Comment (ChasNelson1990) [inline docs/getstarted/createproject.md]: Does not work

File: docs/getstarted/createproject.md [hunk 6]
```diff
@@ -19,20 +19,20 @@ A project can be opened either to the Curate interface, by clicking the **Open X
 
 Now that a project has been created, you can begin to upload data.
 
-![Picture Missing]
+![Project Management Page](/img/manage/manage_projects.png)
```
Comment (ChasNelson1990) [inline docs/getstarted/createproject.md]: Same as top image?

File: docs/getstarted/whatis.md [hunk 7]
```diff
@@ -20,4 +20,6 @@ Tools included:
 
 ![Annotate Screenshot](/img/annotate/annotate.png)
 
-Audit enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary
+[Audit](interfaces/audit) enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary.
+
+![Audit Screenshot]
```
Comment (ChasNelson1990) [inline docs/getstarted/whatis.md]: Does not work?

File: docs/getstarted/interfaces/audit.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 4
+---
+
+# Audit
+
+![Audit Screenshot]
```
Comment (ChasNelson1990) [inline docs/getstarted/interfaces/audit.md]: Does not work

File: docs/getstarted/interfaces/audit.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+- Changes by each user displayed separately, allowing pinpointing of change makers
+
+## Introduction
+
+Welcome to the gliff.ai Audit interface, this has been designed for Team Members to review the curation and annotation of datasets by different users.
+The Audit trail is an immutable record and can be used to demonstrate the robustness of annotation and ML training, both internally and also to external regulators.
+
+## Audit Trail
+
+![Audit Trail]
```
Comment (ChasNelson1990) [inline docs/getstarted/interfaces/audit.md]: Does not work

File: docs/getstarted/interfaces/audit.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+Clicking on any line of the Audit Trail table will open the corresponding record, created by the actions of the named user, at the date and time, relating to the noted image(s).
+Each individual record contains multiple individual actions and details of each, which can be expanded using the Details dropdowns.
+To return to the Audit Trail table, click the **<-** button.
+
+## Search
+
+Using the Search function allows an Audit record to be searched for actions and details.
+
+![Search]
```
Comment (ChasNelson1990) [inline docs/getstarted/interfaces/audit.md]: Does not work

File: docs/misc/defaultlabels.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+To setup default labels, open **Curate** for the relevant project.
+
+![Default Label Pop-Up]
+
+Click the **Set default labels** button and a pop-up should appear.
+By typing in the **New label** box and clicking **+** you can add as many default labels as desired.
+You can then decide, using the check boxes, whether to restrict collaborators (including all active users) to just these labels and whether to allow multiple labels per image.
+Once you have the settings desired, click the **Confirm** button to apply the changes.
+
+_Please Note: In the current version, default labels apply universally to Curate and Annotate._
```
Comment (ChasNelson1990) [inline docs/misc/defaultlabels.md]: Change to:
`_Please note: default labels apply to both Curate and Annotate_`?
Not sure we want to imply that that is going to change?

File: docs/misc/defaultlabels.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+sidebar_position: 1
+---
+
+# Default Labels
+
+Default labels allow team members to pre-load a number of labels in order to simplify the labelling process.
+
+To setup default labels, open **Curate** for the relevant project.
+
+![Default Label Pop-Up]
```
Comment (ChasNelson1990) [inline docs/misc/defaultlabels.md]: Does not work

**Review by ChasNelson1990 — CHANGES_REQUESTED** [review]
I think you forgot to commit the new images?

File: docs/getstarted/adduser.md [hunk 13]
```diff
@@ -29,6 +29,6 @@ To invite a _Team Member_ or _Collaborator_, go to **Manage** and click the **Te
 Tip: Ensure that you are on the right management page (either **Team** or **Collaborators**) before inviting someone to join that user category.
 
 On each page is list of access and control for the user category as above.
```
Comment (RoseBunker) [inline docs/getstarted/adduser.md]: Maybe "On each page there is a list summarising the access and control for the user category."

File: docs/getstarted/interfaces/audit.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+## Features
+
+- Tracks every change made to data within the platform
+- Searchable interface allows for easy change tracking
+- Changes by each user displayed separately, allowing pinpointing of change makers
+
+## Introduction
+
+Welcome to the gliff.ai Audit interface, this has been designed for Team Members to review the curation and annotation of datasets by different users.
```
Comment (RoseBunker) [inline docs/getstarted/interfaces/audit.md]: Team Owners and Team Members

**Review by RoseBunker — DISMISSED** [review]
Where Chas has pointed out that new images haven't appeared, I've not duplicated comments. The last two file changes relating to "package-lock.json" - I marked both as Viewed, but to be honest I don't know what either of those mean.

File: docs/account/accountcustom.md [hunk 15]
```diff
@@ -16,11 +16,13 @@ Your team's storage usage will be revealed here also if you have permissions.
...
+The Team Owner can download individual invoices by clicking **Download** button next to listed invoice.
 Additionally, by clicking the **+** in the top right of the Plan Usage panel, further add ons (such as additional collaborators or storage) can be directly requested.
 
 ## Plug-ins and Custom Code
 
 The platform is capable of integrating custom code that is specific to your team and, as with the whole platform, only visible to your team.
 This works through an API/SDK connection and can be used to affect or analyse images at multiple levels within the platform.
-If you wish to take advantage of this facility please contact us [contact us](https://gliff.ai/contact).
+An example of a gliff.ai compatible plug-in can be found here [geodesic plug-in](https://github.com/gliff-ai/plugin_geodesic-active-contour).
+If you wish to take advantage of this facility please [contact us](https://gliff.ai/contact).
```
Comment (joshuajames-smith) [inline docs/account/accountcustom.md]: `feature` rather than `facility`?

File: docs/basics/firststeps.md [hunk 16]
```diff
@@ -20,11 +20,15 @@ To delete the current annotation in progress, click the **Clear Annotation** but
 To select a different annotation click the **Select** button, click an area of the desired annotation and select the desired functionality to alter.
 To undo the last action, click the **Undo last action** and to redo the action, click the **Redo last action**.
 
-Once annotations have been completed, click the **Save Annotations** button to confirm your changes.
-Please note: if you leave annotate without saving then your annotations will not be added to the image metadata.
```
Comment (joshuajames-smith) [inline docs/basics/firststeps.md:24]: Maybe keep line 24 in for the time being? The saving bug has not been fixed yet (I believe).

**Review by joshuajames-smith — DISMISSED** [review]
I think @ChasNelson1990 and @RoseBunker comments catches most of the changes that would be beneficial, after resolving them and my couple comments I think this is good to go!

**Comment by ChasNelson1990** [comment]
> The last two file changes relating to "package-lock.json" - I marked both as Viewed, but to be honest I don't know what either of those mean.

@RoseBunker - this is totally fine for future reviews too

File: static/img/audit/newfile.md [hunk 17]
```diff
@@ -0,0 +1 @@
+
```
Comment (ChasNelson1990) [inline static/img/audit/newfile.md]: What's this for? Accident perhaps?

File: docs/account/accountcustom.md [hunk 18]
```diff
@@ -16,11 +16,13 @@ Your team's storage usage will be revealed here also if you have permissions.
...
+Clicking on **Billing** will allow the Team Owner to access details of their plan, usage, invoices and payment details.
+The Team Owner can download individual invoices by clicking **Download** button next to listed invoice.
 Additionally, by clicking the **+** in the top right of the Plan Usage panel, further add ons (such as additional collaborators or storage) can be directly requested.
 
 ## Plug-ins and Custom Code
 
 The platform is capable of integrating custom code that is specific to your team and, as with the whole platform, only visible to your team.
 This works through an API/SDK connection and can be used to affect or analyse images at multiple levels within the platform.
-If you wish to take advantage of this facility please contact us [contact us](https://gliff.ai/contact).
+An example of a gliff.ai compatible plug-in can be found here [geodesic plug-in](https://github.com/gliff-ai/plugin_geodesic-active-contour).
```
Comment (ChasNelson1990) [inline docs/account/accountcustom.md]: Can we make this a more natural English sentence?

File: docs/account/accountcustom.md [hunk 19]
```diff
@@ -16,11 +16,13 @@ Your team's storage usage will be revealed here also if you have permissions.
...
+If you wish to take advantage of this feature please [contact us](https://gliff.ai/contact).
```
Comment (ChasNelson1990) [inline docs/account/accountcustom.md:29]: Maybe put this on it's own line?

File: docs/getstarted/whatis.md [hunk 20]
```diff
@@ -20,4 +20,6 @@ Tools included:
 
 ![Annotate Screenshot](/img/annotate/annotate.png)
 
-Audit enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary
+[Audit](interfaces/audit) enables project leaders to track every change or annotation to an image for monitoring and presenting to regulators as necessary.
```
Comment (ChasNelson1990) [inline docs/getstarted/whatis.md]: Should things like "Audit" be "AUDIT"? I know I capitalise stuff but do we have a brand guideline on this? @RoseBunker @joshuajames-smith ?

File: docs/basics/thebasics.md [hunk 21]
```diff
@@ -2,7 +2,7 @@
 sidebar_position: 1
 ---
 
-# Basics
+# Introduction - Basics
```
Comment (ChasNelson1990) [inline docs/basics/thebasics.md]: Maybe just "Introduction"?

File: docs/basics/firststeps.md [hunk 22]
```diff
@@ -15,16 +15,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
```
Comment (ChasNelson1990) [inline docs/basics/firststeps.md:13]: Should this file be renamed to "annotation.md" given that's its menu title?

**Review by ChasNelson1990 — DISMISSED** [review]
Great work.

A couple of screenshots have your open tabs and bookmarks in... can we crop those please?

In fact, maybe we should wrap all fullscreen screenshots in: https://browserframe.com/ or https://www.screely.com/ or similar? @joshuajames-smith  - thoughts on visual brand rules for screenshots? It would prevent the situation where largely empty screens merge into the docs background colour?

What do we think about the example images? I like that we've used medical images but a) are they the right ones, b) should we stick to one type for consistency or vary the image type throughout the docs for diversity?, c) or should we use giraffes and hippos?

Finally, some of the screenshots are huge, e.g. `/basics/assign`, I think we should put in place some rule about sizing - @joshuajames-smith?

File: docs/getstarted/whatis.md (hunk 20 above)
Comment (joshuajames-smith) [reply docs/getstarted/whatis.md]: Brand guidelines are _pending_ but all products should be capitalised, i.e. ANNOTATE, CURATE, AUDIT, MANAGE.

**Comment by joshuajames-smith** [comment]
> What do we think about the example images? I like that we've used medical images but a) are they the right ones, b) should we stick to one type for consistency or vary the image type throughout the docs for diversity?, c) or should we use giraffes and hippos?

I think images may want to be consistent? If we are targeting medical customer at the moment, it may be worth to use this for image genre? It may be worth diversity as we expand but this route way relate more to our customers.

**Review by joshuajames-smith — CHANGES_REQUESTED** [review]
So I think with screenshots we want consistency in treatment.

The size of the screenshots should be limited with: `width: 100%/960px always + height: 480px or less`.

I think the browser wrap is a visual distraction(?) for the purpose of the user documentation but the screenshots do need separated from the background. Edit each screenshot via [Screely](https://www.screely.com/) with: `window: no window + background: #DADADA + padding: 10px 10px`.

Quality of the screenshots are also important. For areas where it is cropped in try zooming in via your browser so the screenshot has a larger size and therefore quality. 
With the above size limits, annotating screenshots could be very valuable. I would use `preview: markup` via Mac with: `shape: rounded square + fill: none + outline: #02FFAD + outline width: 2nd largest`.

Happy to alter anything above if in practice it isn’t working! Possibly worth trialing with one and then grab another review before committing to changing all screenshots.

I've also just become aware of [Scribe](https://scribehow.com/home) which may be a useful to to help build the documentation content?

---

Examples:

Screenshot 2022-05-03 at 16 25 07

Screenshot 2022-05-03 at 16 26 24

**Review by joshuajames-smith — DISMISSED** [review]

File: docs/basics/labels.md [hunk 23]
```diff
@@ -4,13 +4,16 @@ sidebar_position: 8
 
 # Labels
 
-_Tip: To Label and annotation you must first select it, see Annotation Management section of [Annotation: First Steps](firststeps) for further instructions._
+_Tip: To Label and annotation you must first select it, see Annotation Management section of [Annotation](annotation) for further instructions._
```
Comment (RoseBunker) [inline docs/basics/labels.md]: replace 'and' with 'an'

File: docs/basics/annotation.md [hunk 24]
```diff
@@ -15,16 +13,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
 
 ![Controls for the Management of Annotations](/img/annotate/annotate_management.png)
 
+![Annotate Add Button](/img/annotate/annotate_add.png)
```
Comment (RoseBunker) [inline docs/basics/annotation.md:16]: all these images appear broken - not sure if they are or just in my preview

File: docs/basics/annotation.md [hunk 25]
```diff
@@ -15,16 +13,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
...
+![Annotate Undo Button](/img/annotate/annotate_undo.png)
+![Annotate Redo Button](/img/annotate/annotate_redo.png)
 To undo the last action, click the **Undo last action** and to redo the action, click the **Redo last action**.
 
-Once annotations have been completed, click the **Save Annotations** button to confirm your changes.
-Please note: if you leave annotate without saving then your annotations will not be added to the image metadata.
+![Saving Annotation Screen](/img/annotate/annotate_save.png)
+
+At any point, click the **Save Annotations** button to save your progress.
+_Tip: Save regularly as, if you leave annotate or switch images without saving, you will lose any unsaved annotations._
```
Comment (RoseBunker) [inline docs/basics/annotation.md:32]: replace 'annotate' with 'ANNOTATE'

File: docs/basics/annotation.md [hunk 26]
```diff
@@ -15,16 +13,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
...
+
+![Mark as Complete Button location](/img/annotate/annotate_complete.png)
+
+Once annotations have been completed, click the **Mark Annotation As Complete** button.
+This will mark the image as complete for the team members, who can track progress on the **Manage** page.
```
Comment (RoseBunker) [inline docs/basics/annotation.md:37]: replace 'manage' with 'MANAGE'

File: docs/basics/annotation.md [hunk 27]
```diff
@@ -15,16 +13,34 @@ _Tip: To begin annotating an image, double click on the desired image in the cur
...
+_Please note: if you leave annotate without saving then your annotations will not be added to the image metadata._
```
Comment (RoseBunker) [inline docs/basics/annotation.md:38]: replace 'annotate' with 'ANNOTATE'

File: docs/basics/paintbrush.md [hunk 28]
```diff
@@ -37,3 +37,11 @@ Clicking the **Use 3D brushstrokes** button will activate the 3D brush, 3D annot
 The transparency of active and non-active annotations can be altered by clicking the **Annotation Transparency** button.
 Using the two sliders allows you to modify the transparency of both the current (brush) and all other (brush) annotations in the range of 0-100% (opacity).
 This can be a very useful tool to determine the accuracy of drawn annotations by comparing to the background image.
+
+## Strokes vs. Pixels
+
+Brush strokes can be converted from the standard vector-graphic representation into pixel level annotations, based on the size of the pixels of the image being annotated.
+
+![Converter Button](/img/annotate/annotate_strokes.png)
```
Comment (RoseBunker) [inline docs/basics/paintbrush.md:45]: image appears broken in preview

File: docs/basics/thebasics.md [hunk 29]
```diff
@@ -25,7 +25,7 @@ Discover how to take advantage of these features.
 
 A basic form of image annotation, where the user can attach a text label to any or all images in a dataset to categorise them.
 
-## [Annotation: First Steps](firststeps)
+## [Annotation](annotation)
 
 The gliff.ai annotation interface contains a myriad of features and different ways to markup images.
```
Comment (RoseBunker) [inline docs/basics/thebasics.md:30]: replace 'annotate' with 'ANNOTATE'

File: docs/getstarted/adduser.md [hunk 30]
```diff
@@ -28,7 +28,7 @@ To invite a _Team Member_ or _Collaborator_, go to **Manage** and click the **Te
 
 Tip: Ensure that you are on the right management page (either **Team** or **Collaborators**) before inviting someone to join that user category.
```
Comment (RoseBunker) [inline docs/getstarted/adduser.md]: Tip needs italicising to match rest.

File: docs/getstarted/createproject.md [hunk 31]
```diff
@@ -9,7 +9,7 @@ Tip: In order to begin uploading data and annotating images, you must first crea
 ![Project Management Page](/img/manage/manage_projects.png)
 
 In the **Manage** area, click the **Projects** button to access the project management page.
```
Comment (RoseBunker) [inline docs/getstarted/createproject.md:11]: replace 'manage' with 'MANAGE'

File: docs/getstarted/interfaces/audit.md [hunk 32]
```diff
@@ -0,0 +1,36 @@
...
+
+## Features
+
+- Tracks every change made to data within the platform
+- Searchable interface allows for easy change tracking
+- Changes by each user displayed separately, allowing pinpointing of change makers
+
+## Introduction
+
+Welcome to the gliff.ai Audit interface, this has been designed for Team Owners and Team Members to review the curation and annotation of datasets by different users.
```
Comment (RoseBunker) [inline docs/getstarted/interfaces/audit.md]: replace 'audit' with 'AUDIT'

File: docs/getstarted/interfaces/audit.md [hunk 33]
```diff
@@ -0,0 +1,36 @@
...
+The Audit trail is an immutable record and can be used to demonstrate the robustness of annotation and ML training, both internally and also to external regulators.
```
Comment (RoseBunker) [inline docs/getstarted/interfaces/audit.md]: audit with a small 'a'

File: docs/getstarted/interfaces/audit.md [hunk 34]
```diff
@@ -0,0 +1,36 @@
...
+
+![Audit Trail](/img/audit/audit_trail.png)
+
+Clicking on any line of the Audit Trail table will open the corresponding record, created by the actions of the named user, at the date and time, relating to the noted image(s).
+Each individual record contains multiple individual actions and details of each, which can be expanded using the Details dropdowns.
+To return to the Audit Trail table, click the **<-** button.
+
+## Search
+
+Using the Search function allows an Audit record to be searched for actions and details.
```
Comment (RoseBunker) [inline docs/getstarted/interfaces/audit.md]: audit with a small 'a'

File: docs/getstarted/interfaces/audit.md [hunk 35]
```diff
@@ -0,0 +1,36 @@
...
+
+![Search](/img/audit/audit_search.png)
+
+Whilst within an individual Audit record (see above), click the **Search Category** dropdown and select the field to search.
```
Comment (RoseBunker) [inline docs/getstarted/interfaces/audit.md]: audit with a small 'a'

File: docs/misc/defaultlabels.md [hunk 36]
```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 1
+---
+
+# Default Labels
+
+Default labels allow team members to pre-load a number of labels in order to simplify the labelling process.
+
+To setup default labels, open **Curate** for the relevant project.
```
Comment (RoseBunker) [inline docs/misc/defaultlabels.md:9]: replace 'curate' with 'CURATE'

File: docs/misc/defaultlabels.md [hunk 37]
```diff
@@ -0,0 +1,18 @@
...
+
+![Default Label Pop-Up](/img/curate/curate_default.png)
+
+Click the **Set default labels** button and a pop-up should appear.
+By typing in the **New label** box and clicking **+** you can add as many default labels as desired.
+You can then decide, using the check boxes, whether to restrict collaborators (including all active users) to just these labels and whether to allow multiple labels per image.
+Once you have the settings desired, click the **Confirm** button to apply the changes.
+
+_Please note: default labels apply to both Curate and Annotate._
```
Comment (RoseBunker) [inline docs/misc/defaultlabels.md:18]: 'CURATE' and 'ANNOTATE'

**Review by RoseBunker — DISMISSED** [review]
I think the images appear as broken for me in preview, but then when listed as individual image changes they are fine. Only one typo found and suggest capitalising the product names.

File: docs/basics/annotation.md (hunk 24 above)
Comment (philhallbio) [reply docs/basics/annotation.md:16]: They are working in the built preview: https://gray-island-021371003-38.westeurope.1.azurestaticapps.net/docs/basics/annotation

File: docs/basics/annotation.md (hunk 25 above)
Comment (philhallbio) [reply docs/basics/annotation.md:32]: Will do this for all in next update

File: docs/basics/annotation.md (hunk 26 above)
Comment (philhallbio) [reply docs/basics/annotation.md:37]: will do this for all in next update

File: docs/basics/annotation.md (hunk 27 above)
Comment (philhallbio) [reply docs/basics/annotation.md:38]: will do this for all in next update

File: docs/basics/paintbrush.md (hunk 28 above)
Comment (philhallbio) [reply docs/basics/paintbrush.md:45]: working in built preview: https://gray-island-021371003-38.westeurope.1.azurestaticapps.net/docs/basics/paintbrush

File: docs/basics/thebasics.md (hunk 29 above)
Comment (philhallbio) [reply docs/basics/thebasics.md:30]: will do in next update

File: docs/getstarted/createproject.md (hunk 31 above)
Comment (philhallbio) [reply docs/getstarted/createproject.md:11]: will do in next update

File: docs/getstarted/interfaces/audit.md (hunk 32 above)
Comment (philhallbio) [reply docs/getstarted/interfaces/audit.md]: will do in next update

File: docs/misc/defaultlabels.md (hunk 36 above)
Comment (philhallbio) [reply docs/misc/defaultlabels.md:9]: will do in next update

File: docs/misc/defaultlabels.md (hunk 37 above)
Comment (philhallbio) [reply docs/misc/defaultlabels.md:18]: will do in next update

**Review by joshuajames-smith — APPROVED** [review]

**Review by ChasNelson1990 — APPROVED** [review]

**Pull request merged**